teachopencadd/talktorials/019_One_Hot_Encoding/talktorial.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Change the title to <code>The *One-Hot Encoding* concept</code>, because <code>T019</code>   is unsuitable for html webpage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Since every notebook has a talktorial number, so for the unity across the notebooks I have included that talktorial number. Once the final number is decided we can change it later ??</p>

Reply:
<p>Oh, It's Ok for <code>T019</code>.  In the lasted version, all the title have talktorial number. Sorry for the mistake.</p>